// containers/watch/index.js
import React, { Component } from 'react'
import {connect} from 'react-redux'
import R from 'ramda'
import {Link} from 'react-router'

import {fetchWatchById, addWatchToCart} from 'actions'

import { getWatchById} from 'selectors'


import BasketCart from 'components/BasketCart'

class Watch extends Component {
    componentDidMount () {
        this.props.fetchWatchById(this.props.params.id)
    }

    renderFields () {
        const {watch} = this.props
        const columnFields = R.compose(
            R.toPairs,
            R.pick([
                'collection',
                'mechanism',
                'glass',
                'color',
                'bracelet',
                'case',
                'resistant'
            ])
        )(watch)
        
        return columnFields.map(([key,value]) => (
            <div className='column' key={key}>
                <div className='ab-details-title'>
                    <p>{key}</p>
                </div>
                <div className='ab-details-info'>
                    <p>{value}</p>
                </div>
            </div>
        ))
    }

    renderContent () {
        const {watch} = this.props
        
        return (
            <div className='thumbnail'>
                <div className='row'>
                    <div className='col-md-6'>
                        <img className='img-thumbnail mini-image' 
                             src={watch.image} 
                             alt={watch.name}
                             
                        />
                    </div>
                    <div className='col-md-6'>
                        {this.renderFields()}
                    </div>
                </div>
                <div className='caption-full'>
                    <h4 className='pull-right'>${watch.price}</h4>
                    <h4>{watch.name}</h4>
                    <p>{watch.description}</p>
                </div>
            </div> 
        )
    }

    renderSidebar () {
        const {watch, addWatchToCart} = this.props
        return (
            <div> 
                <p className='lead'> Quick Shop  </p>
                <BasketCart />
                <div className='form-group'>
                    <h1>{watch.name}</h1>
                    <h2>${watch.price}</h2>
                </div>
                <Link 
                    to='/' 
                    className='btn btn-info btn-block'
                >
                    Back to store
                </Link>
                <button 
                    type='button'
                    className='btn btn-success btn-block'
                    onClick={() => addWatchToCart(watch.id)}
                >
                    Add to Cart
                </button>
            </div>
        )
    }

    render() {
        const {watch} =this.props
        return (
            <div className='view-container'>
                <div className='container'>
                    <div className='row'>
                        <div className='col-md-9'>
                            {watch && this.renderContent()}
                        </div>
                        <div className='col-md-3'>
                            {watch && this.renderSidebar()}
                        </div>
                    </div>            
                </div>        
            </div>
        )
    }
}

const mapStateToProps = state => ({
    watch: getWatchById(state, state.watchPage.id)
})

const mapDispatchToProps = {
    fetchWatchById,
    addWatchToCart
}


export default connect(mapStateToProps,mapDispatchToProps)(Watch)
