// api/mockWatches.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "JUST WHITE",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 33,
    image: "/uploads/142528_gw151.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "white",
    bracelet: "plastic",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "2",
    categoryId: "1",
    name: "WHITE BISHOP",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 47,
    image: "/uploads/142528_gw164.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "white",
    bracelet: "silicone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "3",
    categoryId: "1",
    name: "ROSE EXPLOSION",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 44,
    image: "/uploads/142528_gz291.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "multicolored",
    bracelet: "silicone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "4",
    categoryId: "1",
    name: "SOUL2WATCH",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 39,
    image: "/uploads/142528_lk333.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "white",
    bracelet: "plastic",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "5",
    categoryId: "2",
    name: "BLACK CLASSINESS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 74,
    image: "/uploads/142528_sfk361.jpg",
    collection: "Skin",
    mechanism: "quartz",
    glass: "organic",
    color: "black",
    bracelet: "silicone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "6",
    categoryId: "2",
    name: "WHITE SILK",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 64,
    image: "/uploads/142528_sfk374g.jpg",
    collection: "Skin",
    mechanism: "quartz",
    glass: "organic",
    color: "gold",
    bracelet: "textile",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "7",
    categoryId: "2",
    name: "SEMINATO",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 85,
    image: "/uploads/142528_sfw107.jpg",
    collection: "Skin",
    mechanism: "quartz",
    glass: "organic",
    color: "gold",
    bracelet: "textile",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "8",
    categoryId: "4",
    name: "DIGITAL ROSE",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 124,
    image: "/uploads/142528_surp100.jpg",
    collection: "Digital",
    mechanism: "quartz",
    glass: "organic",
    color: "rose",
    bracelet: "silicone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "9",
    categoryId: "3",
    name: "ROSETA BIANCA",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 91,
    image: "/uploads/142528_yas109.jpg",
    collection: "Irony",
    mechanism: "auto",
    glass: "organic",
    color: "white",
    bracelet: "leather",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "10",
    categoryId: "3",
    name: "SINCE 2013",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 83,
    image: "/uploads/142528_ycs571.jpg",
    collection: "Irony",
    mechanism: "quartz",
    glass: "organic",
    color: "blue",
    bracelet: "leather",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "11",
    categoryId: "3",
    name: "TRESOR BLANC",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 60,
    image: "/uploads/142528_yls141gc.jpg",
    collection: "Irony",
    mechanism: "quartz",
    glass: "organic",
    color: "white",
    bracelet: "steel 316L",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "12",
    categoryId: "3",
    name: "MOON SHINE",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 55,
    image: "/uploads/142528_ywg400.jpg",
    collection: "Irony",
    mechanism: "quartz",
    glass: "organic",
    color: "blue",
    bracelet: "leather",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "13",
    categoryId: "3",
    name: "APPIA",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 79,
    image: "/uploads/142528_yws401.jpg",
    collection: "Irony",
    mechanism: "quartz",
    glass: "organic",
    color: "white",
    bracelet: "leather",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "14",
    categoryId: "1",
    name: "CENTRINO",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 34,
    image: "/uploads/144221_suos104.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "blue",
    bracelet: "silikone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "15",
    categoryId: "3",
    name: "NUMMER 100",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 78,
    image: "/uploads/144221_susw405.jpg",
    collection: "Irony",
    mechanism: "quartz",
    glass: "organic",
    color: "gray",
    bracelet: "textile",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "16",
    categoryId: "1",
    name: "BLACK SUIT",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 38,
    image: "/uploads/145701_gb247r.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "black",
    bracelet: "silikone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "17",
    categoryId: "1",
    name: "THE STRAPPER",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 29,
    image: "/uploads/145701_gb289.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "black",
    bracelet: "silikone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "18",
    categoryId: "1",
    name: "BLACK SUIT ORANGE",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 55,
    image: "/uploads/145701_gb754.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "black",
    bracelet: "plastic",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "19",
    categoryId: "1",
    name: "SOMETHING NEW",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 47,
    image: "/uploads/145701_lb153.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "black",
    bracelet: "plastic",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "20",
    categoryId: "3",
    name: "BODY AND SOUL",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 69,
    image: "/uploads/145701_yls186g.jpg",
    collection: "Irony",
    mechanism: "auto",
    glass: "organic",
    color: "silver",
    bracelet: "steel 316L",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "21",
    categoryId: "1",
    name: "COLOR THE SKY",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 77,
    image: "/uploads/148179_gs124.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "colorful",
    bracelet: "plastic",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "22",
    categoryId: "3",
    name: "QUITNESS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 101,
    image: "/uploads/149666_svom100m.jpg",
    collection: "Irony",
    mechanism: "auto",
    glass: "organic",
    color: "metal",
    bracelet: "steel 316L",
    case: "steel 316L",
    resistant: "30 meters"
  },
  {
    id: "23",
    categoryId: "2",
    name: "SKINNOIR",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 97,
    image: "/uploads/149666_svub100.jpg",
    collection: "Skin",
    mechanism: "quartz",
    glass: "organic",
    color: "black",
    bracelet: "silikone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "24",
    categoryId: "1",
    name: "SEA POOL",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 31,
    image: "/uploads/151221_gl700.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "blue",
    bracelet: "silikone",
    case: "plastic",
    resistant: "30 meters"
  },
  {
    id: "25",
    categoryId: "1",
    name: "MEANT LEAVE",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 44,
    image: "/uploads/151221_svul100.jpg",
    collection: "Originals",
    mechanism: "quartz",
    glass: "organic",
    color: "multicolor",
    bracelet: "plastic",
    case: "plastic",
    resistant: "30 meters"
  }
]
